crates/ruff_linter/src/rules/flake8_async/rules/sleep_forever_call.rs

```diff
@@ -0,0 +1,109 @@
+use ruff_diagnostics::{Diagnostic, Edit, Fix, FixAvailability, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::{Expr, ExprCall, ExprNumberLiteral, Number};
+use ruff_python_semantic::Modules;
+use ruff_text_size::Ranged;
+
+use crate::{checkers::ast::Checker, importer::ImportRequest};
+
+/// ## What it does
+/// Checks for uses of `trio.sleep()` with >24 hour interval.
+///
+/// ## Why is this bad?
```

Comment:
We were discussing how "no gil" was renamed to "free threading" to remove negative language.

Thoughts on changing "Why is this bad" to something like "How could it be better?" across all rules?